src/web/Router.js

```diff
@@ -7,18 +7,96 @@ import Layout from "./theme/ui/templates/Layout";
 import Header from "./theme/modules/Header";
 import Footer from "./theme/modules/Footer";
 import Home from "./theme/pages/Home";
-import Product from "./theme/pages/Product";
-import Checkout from "./theme/pages/Checkout";
+import TitleWithDescription from "theme/ui/molecules/TitleWithDescription";
+import { H2 } from "theme/ui/atoms/Typography/Heading";
+import OnePage from "theme/pages/OnePage";
 import NotFound from "./theme/pages/NotFound";
 
+const Cards = [
+  {
+    name: "our-selection",
+    style: "light-blue",
+    contentStyle: "card",
+    title: (
+      <TitleWithDescription
+        title={<H2>Our selection</H2>}
+        description={
+          <div className="our-selection__title-description">
+            <p>
+              Sometimes, you don’t have time to search and want instant results.
+              Be sure to regularly check our Cultural Sites’ Selection.{" "}
```

Comment:
I dont think the extra space at the end is needed here if the following text is on another line ?